api/client/stats.go

```diff
@@ -139,7 +139,7 @@ func (s *containerStats) Display(w io.Writer) error {
 //
 // This shows real-time information on CPU usage, memory usage, and network I/O.
 //
-// Usage: docker stats [OPTIONS] [CONTAINER...]
+// Usage: docker stats [OPTIONS] CONTAINER [CONTAINER...]
```

Comment:
OK, So that's to say the tips that provided by the command "docker stats --help"  isn't up to date yet. I will cancel this modification, and try to solve this in other branch, thank you for reminding me this !